core/src/main/java/net/opentsdb/query/pojo/TagVFilter.java

```diff
@@ -660,6 +660,14 @@ public TagVFilter build() {
             "The tagk cannot be null or empty");
       }
       
+      // V2 librange conversion
+      if (type.equals(TagVLiteralOrFilter.FILTER_NAME)) {
+        if (filter.startsWith("librange://@")) {
```

Comment:
can this be handled in our plugin?

Reply:
Yes, but it won't be clean. Several classes need to be extended and will introduce a bit more plugins as we discussed.